cpp/daal/include/data_management/data/internal/numeric_table_sycl_homogen.h

```diff
@@ -338,6 +343,7 @@ class DAAL_EXPORT SyclHomogenNumericTable : public SyclNumericTable
         static services::Status read(const services::internal::Buffer<U> & buffer, BlockDescriptor<T> & block, size_t nRows, size_t nCols)
         {
             DAAL_ASSERT(buffer.size() == nRows * nCols);
```

Comment:
```suggestion
            DAAL_OVERFLOW_CHECK_BY_MULTIPLICATION(size_t, nRows, nCols);
            DAAL_ASSERT(buffer.size() == nRows * nCols);
```

Reply:
I'm not addressing SDL in data management now